nada_dsl/ast_util.py

```diff
@@ -113,7 +113,7 @@ class IfElseASTOperation(ASTOperation):
     arg_0: int
     arg_1: int
 
-    def inner_operations(self):
+    def child_operations(self):
         return [self.this, self.arg_0, self.arg_1]
```

Comment:
I will rename `this` to `condition`, even being peaky, `condition_child`, `true_branch_child`, `false_branch_child`